src/components/Asset/AssetActions/Trade/Output.tsx

```diff
@@ -28,28 +30,28 @@ export default function Output({
 
   // Get swap fee
   useEffect(() => {
-    if (!poolAddress || !isAssetNetwork) return
+    if (!poolInfo || !isAssetNetwork) return
 
     async function getSwapFee() {
```

Comment:
1. nothing `async` going on in here anymore so no need to wrap what's happening in this effect in a method.
2. `poolInfo` might change a lot but effect only makes use of `poolInfo.poolFee` so better use `poolInfo?.poolFee` in the dependency array, and at top `if (!poolInfo?.poolFee ...)...`
3. don't think we need `poolAddress` then in the dependency array as we are not using it anymore in this effect